src/components/Header/Header.jsx

```diff
@@ -1,13 +1,17 @@
-import Navbar from "../Navbar/Navbar";
+import { useContext } from "react";
+import Navbar from '../Navbar/Navbar';
+import { AuthContext } from "../../containers/AuthContext";
 // import styles from './Header.module.scss';
 
 export default function Header() {
-  return (
-    <header>
-      <div className="container">
-        <p>Header </p>
-        <Navbar />
-      </div>
-    </header>
-  );
+    const authContext = useContext(AuthContext);
+    console.log(authContext.token);
```

Comment:
oops, forgot to remove redundant console.log.
if this is needed for development purpose then we have 2 other options:
- we can always debug it with Dev Tools in browser. Works perfect in local.
- console.debug() + logging configuration: if logging available - we do console debug. otherwise in production we do not console log it.

Reply:
Deleted